pkg/parser/utils.go

```diff
@@ -71,34 +72,47 @@ func extractDescriptionData(doc string) DescriptionData {
 	var exampleLines []string
 	var notesLines []string
 	var deprecationNoteLines []string
+	var returnsLines []string
 
 	var description string
 	var example string
 	var notes string
 	var deprecationNote string
+	var returns string
 
 	isExample := false
 	isNotes := false
 	isDeprecationNote := false
+	isReturns := false
 
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
 		if strings.HasPrefix(line, "Example:") {
 			isExample = true
 			isNotes = false
 			isDeprecationNote = false
+			isReturns = false
 			continue
 		}
 		if strings.HasPrefix(line, "Notes:") {
 			isNotes = true
 			isExample = false
 			isDeprecationNote = false
+			isReturns = false
 			continue
 		}
 		if strings.HasPrefix(line, "Deprecated:") {
 			isDeprecationNote = true
 			isExample = false
 			isNotes = false
+			isReturns = false
+			continue
+		}
+		if strings.HasPrefix(line, "Returns:") {
```

Comment:
this should probably be some specific keyword, like `@returns` or `$returns`

Reply:
agreed, i'd prefer `@returns`

Reply:
Go (Godoc) does not have a proper word for that. If we want to keep consistency across Pallas, we should use:
```
// random description
//
// Returns:
// ...
```

Reply:
Is compliance with godoc necessary? I don't think people would want to use godoc and pallas at the same time

Reply:
Actually I agree with mirko, just doing `Returns:` and `Deprecated:` is better

Reply:
something like what's done in libadwaita/gtk https://gitlab.gnome.org/GNOME/libadwaita/-/blob/main/src/adw-about-window.c#L3406-3457

Reply:
Ok, I am fine with it.
Let’s leave them as they are.